src/components/tax-rates/TaxRateSelector.tsx

```diff
@@ -45,7 +45,7 @@ export function TaxRateSelector(props: Props) {
           props.onChange && props.onChange(record)
         }
         value="rate"
-        defaultValue={props.defaultValue === 0 ? null : props.defaultValue}
```

Comment:
@beganovich Please let me know if there is a specific reason for this condition that we had, or maybe it was something required by an older version of the Combobox component.